resources/views/components/achievement/thin-type-indicator.blade.php

```diff
@@ -1,13 +1,15 @@
-@props([
-    'type' => null, // AchievementType
-])
-
 <?php
 
 use App\Platform\Enums\AchievementType;
 use App\Enums\UserPreference;
 use Illuminate\Support\Facades\Auth;
+?>
 
+@props([
+    'type' => null, // AchievementType
+])
```

Comment:
I looked into https://github.com/TomasVotruba/bladestan but it doesn't seem to work well with props. I'll keep it on my list to evaluate and see what it can do for us.